### PR TITLE
Adds util functions and changes configs

### PR DESCRIPTION
**Pull request by Ajeyakrishna-k**
Date: `11 Dec 2023`

Developer Name: @Ajeyakrishna-k 

----

## Issue Ticket Number:- 
- #19

## Description: 
- Refactor the way configs are handled
- Adds an array util function
- Adds util function to generate discord jwt token

Is Under Feature Flag 
- [ ] Yes
- [x] No

Database changes
- [ ] Yes
- [x] No

Breaking changes (If your feature is breaking/missing something please mention pending tickets)
- [ ] Yes
- [x] No

Is Development Tested?

- [x] Yes
- [ ] No

### Add relevant Screenshot below ( e.g test coverage etc. )

Screenshot 2023-12-11 at 7 53 25 PM

### CODE REVIEW

File: src/utils/generateJwt.ts [hunk 1]
```diff
@@ -18,3 +18,18 @@ export const generateJwt = async (env: env) => {
 		throw new Error('Error in generating the auth token');
 	}
 };
+
+export const generateDiscordBotJwt = async (env: env) => {
+	try {
+		const authToken = await jwt.sign(
+			{
+				exp: Math.floor(Date.now() / 1000) + 2,
```
Comment (heyrandhirs) [inline src/utils/generateJwt.ts:26]: can you please explain why have we added + 2

File: src/constants/commons.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const MISSED_UPDATES_PROD_ROLE_ID = '1183553844811153458';
+export const MISSED_UPDATES_STAGING_ROLE_ID = '1183553844811153458';
+export const MISSED_UPDATES_DEVELOPMENT_ROLE_ID = '1181214205081296896';
```
Comment (heyrandhirs) [inline src/constants/commons.ts:3]: should this be in config instead of constants ?

File: wrangler.toml [hunk 3]
```diff
@@ -3,19 +3,24 @@ main = "src/worker.ts"
...
 kv_namespaces = [
 	{ binding = "CronJobsTimestamp", id = "3a10f726c95d4afea9dee5fd00f029b9" }
 ]
+services = [
+  { binding = "DISCORD_BOT", service = "discord-slash-commands", environment = "production" }
+]
 
 [triggers]
-crons = ["0 */4 * * *", "0 */6 * * *"]
+crons = ["0 */4 * * *", "0 */6 * * *","0 */12 * * *" ]
```
Comment (heyrandhirs) [inline wrangler.toml:23]: didn't get this line. can you please explain what is being updated here

**Review by heyrandhir — CHANGES_REQUESTED** [review]

File: src/constants/commons.ts (hunk 2 above)
Comment (Ajeyakrishna-k) [reply src/constants/commons.ts:3]: Do you mean as environment variable?

File: src/utils/generateJwt.ts (hunk 1 above)
Comment (Ajeyakrishna-k) [reply src/utils/generateJwt.ts:26]: Token is valid only for 2 seconds

File: wrangler.toml (hunk 3 above)
Comment (Ajeyakrishna-k) [reply wrangler.toml:23]: Adds a cron tab entry for every 12hr

File: src/constants/commons.ts (hunk 2 above)
Comment (heyrandhirs) [reply src/constants/commons.ts:3]: Yeah right

Comment (Ajeyakrishna-k) [reply src/constants/commons.ts:3]: This is not a secret. 
- Anyone within our discord server can access it.
- Anyone outside have no use for it

Comment (Ajeyakrishna-k) [reply src/constants/commons.ts:3]: Its also done the same way in our backend repo:
https://github.com/Real-Dev-Squad/website-backend/blob/develop/config/production.js

**Review by heyrandhir — APPROVED** [review]

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**